server/middleware.js

```diff
@@ -1,3 +1,5 @@
+var globalRequestId = 0;
```

Comment:
This will not work as we have 3 ssr processes running simultaneously in production.
We need something like [nanoid](https://github.com/ai/nanoid).

Please also consider [express-id-log-decorator](https://github.com/itross/express-id-log-decorator).
As we want every log line produced in any flow to print its request_id